Fix bug in TenableSC context (#18146)
demisto/content: Packs/Tenable_sc/ReleaseNotes/1_0_5.md

#### Integrations
##### Tenable.sc

> - Fix bug in tenable-sc-get-vulnerability which caused context to be overwritten

Reviewer: ```suggestion
- Fixed an issue where the ***tenable-sc-get-vulnerability*** command caused the context to be overwritten.
```
